### PR TITLE
Added new third-party tools - Airbyte 

### LINKED ISSUE #59: Add new third-party tools
We're eager to grow our ecosystem.

To help with this, we can expand [our docs around third-party tools](https://questdb.io/docs/third-party-tools/overview/).

If you're aware of a tool which can interface with QuestDB, but does not have a 3rd party tools page, please raise it in this issue. We'll respond as soon as we can, and give the thumb ups. After that, create a PR to include it based on the current pattern, and that's it!

Also, auditing the current third-party tools page & confirming they're all in the overview is a helpful task.

### PR DESCRIPTION
**Pull request by abhirajadhikary06**
Overview Section: Added Airbyte as a new third-party tool in the documentation under the Data Ingestion and Streaming section. The overview includes a description of how Airbyte can be leveraged with QuestDB for seamless data ingestion.

User Guide: Detailed step-by-step guide on configuring and using Airbyte to transfer data into QuestDB. The guide includes code snippets, best practices, and troubleshooting tips for smooth integration.

Adds to #59

### CODE REVIEW

**Comment by abhirajadhikary06** [comment]
@goodroot I request you to check this and merge it

**Comment by abhirajadhikary06** [comment]
@goodroot Can you review and if all's ok then merge this please.

File: third-party-tools/airbyte.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+- [Verifying the Integration](#verifying-the-integration)
+- [Best Practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+- [Summary](#summary)
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- **QuestDB** must be running and accessible. Check out the [QuestDB quick start guide](https://questdb.io/docs/get-started/docker/).
```
Comment (goodroot) [inline third-party-tools/airbyte.md:32]:
```suggestion
- **QuestDB** must be running and accessible. Check out the [QuestDB quick start guide](/docs/get-started/docker/).
```

File: third-party-tools/airbyte.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+- **Airbyte** installed using Docker, or locally via [Homebrew](https://brew.sh/) on macOS. For detailed installation instructions, refer to the [official Airbyte documentation](https://docs.airbyte.com/deploying-airbyte/local-deployment).
+
+## Configuring Airbyte
+
+Airbyte uses a user-friendly interface for configuration, allowing you to easily set up your sources and destinations. Once Airbyte is running, navigate to the Airbyte dashboard at `http://localhost:8000`.
+
+### Adding a Source
```
Comment (goodroot) [inline third-party-tools/airbyte.md:39]:
```suggestion
### Adding a source
```

File: third-party-tools/airbyte.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+1. Click on the **Sources** tab.
+2. Select the desired source from the list of available connectors.
+3. Fill in the necessary configuration details, such as connection strings, credentials, and any other required fields.
+
+### Adding a Destination
```
Comment (goodroot) [inline third-party-tools/airbyte.md:45]:
```suggestion
### Adding a destination
```

File: third-party-tools/airbyte.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+1. Click on the **Destinations** tab.
+2. Select **QuestDB** as your destination.
+3. Enter the required connection details for QuestDB:
+   - **Host**: `localhost` or your QuestDB server IP
+   - **Port**: `8812`
+   - **User**: `admin`
+   - **Password**: (Leave empty if using default settings)
+
+### Generating Configuration
```
Comment (goodroot) [inline third-party-tools/airbyte.md:55]:
```suggestion
### Generating configuration
```

File: third-party-tools/airbyte.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+Airbyte allows you to test your configuration after entering the details. Click on **Test Connection** to ensure that the connection to the source and destination is successful. Once confirmed, you can proceed to create a connection between the source and destination.
+
+## Example Inputs
```
Comment (goodroot) [inline third-party-tools/airbyte.md:59]:
```suggestion
## Example inputs
```

File: third-party-tools/airbyte.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+
+In this section, we will demonstrate how to set up Airbyte to extract data from a PostgreSQL source and send it to QuestDB.
+
+### PostgreSQL Source Configuration
```
Comment (goodroot) [inline third-party-tools/airbyte.md:63]:
```suggestion
### PostgreSQL source configuration
```

File: third-party-tools/airbyte.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+
+```sql
+SELECT * FROM your_table LIMIT 10;
+```
+
+### Writing to QuestDB
+
+After configuring your PostgreSQL source, the next step is to configure the destination to send the extracted data to QuestDB.
+
+#### QuestDB Destination Configuration
```
Comment (goodroot) [inline third-party-tools/airbyte.md:94]:
```suggestion
#### QuestDB destination configuration
```

File: third-party-tools/airbyte.md [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+  },
+  {
+    "column1": "value2",
+    "column2": 456,
+    "column3": "2023-10-19T12:05:00Z"
+  }
+]
+```
+
+### Running the Connection
```
Comment (goodroot) [inline third-party-tools/airbyte.md:132]:
```suggestion
### Running the connection
```

File: third-party-tools/airbyte.md [hunk 9]
```diff
@@ -0,0 +1,306 @@
...
+  "destinationId": "your_questdb_destination_id",
+  "syncMode": "full_refresh",
+  "schedule": {
+    "units": 1,
+    "timeUnit": "hours"
+  }
+}
+```
+
+### Monitoring Sync Status
```
Comment (goodroot) [inline third-party-tools/airbyte.md:151]:
```suggestion
### Monitoring sync status
```

File: third-party-tools/airbyte.md [hunk 10]
```diff
@@ -0,0 +1,306 @@
...
+
+Once the sync process is initiated, you can monitor its status directly in the Airbyte dashboard.
+
+#### Example Sync Status Output
```
Comment (goodroot) [inline third-party-tools/airbyte.md:155]:
```suggestion
#### Example sync status output
```

File: third-party-tools/airbyte.md [hunk 11]
```diff
@@ -0,0 +1,306 @@
...
+{
+  "status": "COMPLETED",
+  "records_transferred": 150,
+  "start_time": "2023-10-19T12:00:00Z",
+  "end_time": "2023-10-19T12:10:00Z",
+  "errors": []
+}
+```
+
+### Debugging Tips
```
Comment (goodroot) [inline third-party-tools/airbyte.md:167]:
```suggestion
### Debugging tips
```

File: third-party-tools/airbyte.md [hunk 12]
```diff
@@ -0,0 +1,306 @@
...
+
+If you encounter issues during the sync process, consider the following debugging steps:
+
+- **Check Connection Settings**: Ensure the host, port, and authentication details are correct in both the source and destination configurations.
+- **Review Logs**: Check the Airbyte logs for any error messages. Logs can provide insight into connection failures or data format issues.
+- **Test Queries**: Use the query tools available in your PostgreSQL and QuestDB interfaces to test individual queries and see what data is being extracted or sent.
+
+## Verifying the Integration
```
Comment (goodroot) [inline third-party-tools/airbyte.md:175]:
```suggestion
## Verifying the integration
```

File: third-party-tools/airbyte.md [hunk 13]
```diff
@@ -0,0 +1,306 @@
...
+5. **Sample Memory Query Results**:
+
+```plaintext
+| timestamp            | total      | available | used        | free        |
+|----------------------|------------|-----------|-------------|-------------|
+| 2024-10-19T12:00:00Z | 8000       | 3000      | 4000        | 1000        |
+| 2024-10-19T12:05:00Z | 8000       | 2900      | 4100        | 1000        |
+```
+
+## Best Practices
```
Comment (goodroot) [inline third-party-tools/airbyte.md:230]:
```suggestion
## Best practices
```

File: third-party-tools/airbyte.md [hunk 14]
```diff
@@ -0,0 +1,306 @@
...
+If you encounter issues during the sync process, consider the following debugging steps:
+
+- **Check Connection Settings**: Ensure the host, port, and authentication details are correct in both the source and destination configurations.
+- **Review Logs**: Check the Airbyte logs for any error messages. Logs can provide insight into connection failures or data format issues.
+- **Test Queries**: Use the query tools available in your PostgreSQL and QuestDB interfaces to test individual queries and see what data is being extracted or sent.
+
+## Verifying the Integration
+
+1. **Access the QuestDB Web Console**:  
+   Navigate to the [QuestDB Web Console](https://questdb.io/docs/develop/web-console/) at `http://127.0.0.1:9000/`. Once you're on the console, check the Schema Navigator in the top left. You should see two new tables:
```
Comment (goodroot) [inline third-party-tools/airbyte.md:178]:
```suggestion
   Navigate to the [QuestDB Web Console](/docs/develop/web-console/) at `http://127.0.0.1:9000/`. Once you're on the console, check the Schema Navigator in the top left. You should see two new tables:
```

File: third-party-tools/airbyte.md [hunk 15]
```diff
@@ -0,0 +1,306 @@
...
+
+2. **Run a Query on the `cpu` Table**:  
+   To verify that data is being correctly written to the `cpu` table, type the following query in the editor and click **RUN**:
+
+```sql
+SELECT * FROM cpu;
+```
+
+3. **Inspect the Results**:  
+   After running the query, the results should display columns for each metric collected by the Telegraf plugin for monitoring CPU usage, such as:
```
Comment (goodroot) [inline third-party-tools/airbyte.md:191]:
```suggestion
   After running the query, the results should display columns for each metric collected by the [Telegraf](/docs/third-party-tools/telegraf/) plugin for monitoring CPU usage, such as:
```

File: third-party-tools/airbyte.md [hunk 16]
```diff
@@ -0,0 +1,306 @@
...
+- **Configuring Airbyte**: Set up Airbyte to connect with your desired data sources and QuestDB as the destination.
+- **Data Validation**: Verify data integrity post-load and ensure the expected records are ingested.
+- **Best Practices**: Optimize data loads, monitor resource utilization, and use indexing for efficient querying.
+
+For further details and resources, refer to the following links:
+
+- [Airbyte Documentation](https://docs.airbyte.com/)
+- [QuestDB Documentation](https://questdb.io/docs/)
+- [QuestDB Web Console Guide](https://questdb.io/docs/develop/web-console/)
+- [Airbyte GitHub Repository](https://github.com/airbytehq/airbyte)
```
Comment (goodroot) [inline third-party-tools/airbyte.md:306]:
```suggestion
- [Airbyte Documentation](https://docs.airbyte.com/)
- [QuestDB Web Console Guide](/docs/develop/web-console/)
- [Airbyte GitHub Repository](https://github.com/airbytehq/airbyte)
```

**Review by goodroot — APPROVED** [review]

**Pull request merged**